roles/lib_openshift/src/class/oc_project.py

```diff
@@ -0,0 +1,183 @@
+# pylint: skip-file
+# flake8: noqa
+
+
+# pylint: disable=too-many-instance-attributes
+class OCProject(OpenShiftCLI):
+    ''' Class to wrap the oc command line tools '''
+    kind = 'namespace'
```

Comment:
We should probably get some confirmation that this wouldn't introduce some unintended behavior.

Since `oc adm new-project` is being used to create the project it might be fine. I'm not sure if there would be any other project-specific things that would effect update/delete.

@liggitt @deads2k can either of you confirm if there are any edge cases with update/delete for a project through namespace would cause issues?